interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -710,6 +710,19 @@ public void gracefulShutdown() throws Exception {
     verifyNoMoreInteractions(responseObserver);
   }
 
+  /** Sends an rpc to an unimplemented method on the server. */
```

Comment:
That seems it would lead to a random ordering of tests. Keeping the same order as the test descriptions seems more useful, but I guess you would mainly be favoring mostly ignoring the order.